docs/metadata.rst

```diff
@@ -0,0 +1,98 @@
+Import metadata using the Command Line Interface (CLI)
+=======================================================
+
+Description:
+------------
+
+This chapter will show how to import metadata starting from a local CSV file and ending with OMERO.tables on images or Key-Value pairs on images using the Command Line Interface (CLI) and the OMERO server-side scripts.
+
+This action is typically done after a successful import of images.
+
+We will show:
+
+-  How to import metadata from local CSV file in a bulk manner and turn them into OMERO.tables on images using CLI
+
+- How to turn the OMERO.tables on images into Key-Value pair annotations on images in bulk manner using CLI
+
+- How to import metadata from local CSV file and use a server-side script in OMERO to turn these into OMERO.tables on images
+
+- How to construct a simple file to turn the metadata stored in OMERO.tables into Key-Value pairs on images using CLI
+
+**Resources:**
+--------------
+
+-  Documentation:
+
+   -  https://docs.openmicroscopy.org/latest/omero/users/cli/installation.html
+
+   -  `https://docs.openmicroscopy.org/omero/latest/users/cli/index.html <https://docs.openmicroscopy.org/omero/latest/users/cli/index.html>`__
+
+-  Data: example images from
+
+   -  IDR data http://idr.openmicroscopy.org/webclient/?show=project-51
+
+-  Metadata plugin for OMERO
+
+   - https://pypi.org/project/omero-metadata/
+
+-  Bulkmap config yml files defining the various Key-Value pairs parameters, such as the groups and other parameters.
+
+   - https://github.com/IDR/idr0021-lawo-pericentriolarmaterial/blob/9479af85f19487f215e3dfdd31a1b587370ed3cf/experimentA/idr0021-experimentA-bulkmap-config.yml
+
+-  Annotation csv file defines the content of OMERO.tables for each image.
+
+   - https://github.com/IDR/idr0021-lawo-pericentriolarmaterial/blob/9479af85f19487f215e3dfdd31a1b587370ed3cf/experimentA/idr0021-experimentA-annotation.csv
+
+Setup:
+------
+
+**Metadata plugin installation**
+
+- Go to the environment where you installed your OMERO.cli as specified under -  https://docs.openmicroscopy.org/latest/omero/users/cli/installation.html.
+
+- Activate the virtual environment.
+
+- Run::
+    
+    $ pip install omero-metadata
+
+**Step-by-step:**
+-----------------
+
+#.  On your local machine, open a terminal
+
+#.  Activate the virtual environment where ``omero-py`` is installed or add it to ``PATH`` e.g.::
+
+    $ export PATH=/opt/omero/server/venv3/bin:$PATH
+
+#.  Download the csv from https://github.com/IDR/idr0021-lawo-pericentriolarmaterial/blob/9479af85f19487f215e3dfdd31a1b587370ed3cf/experimentA/idr0021-experimentA-annotation.csv 
+
+#.  The variable ``$ID​`` below is the ID of the ​Project, in this example case it is the Project containing the idr0021 study. To add annotations from a local csv file to the images in the said Project in the form of OMERO.tables, run::
```

Comment:
This assumes the user has imported `idr0021` somewhere, but I can't see any mention of it in https://omero-guides.readthedocs.io/en/latest/upload/docs/import.html

Reply:
No, but the data are mentioned in this very guide, under Resources. The thinking is - if you are in our workshop, you will have the data there ready for you, so no worries. If you are outwith the workshop, you will probably not want to be importing idr0021 on your server. just to test the workflows.  You will want to have the files such as bulkmap and tsv and compare the patterns there with the data via navigating to idr.